source/common/CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ add_library(
   grpc/common.cc
   grpc/http1_bridge_filter.cc
   grpc/rpc_channel_impl.cc
+  grpc/utility.cc
```

Comment:
nit: I think just add this code to common.cc, probably don't need a new source file